frontend/src/app/docs/api-docs/api-docs.component.html

```diff
@@ -218,17 +218,41 @@ <h3 *ngIf="( item.type === 'category' ) && ( item.showConditions.indexOf(network
-  <p>Then let <span class='math'>r</span> be the number of removed transactions—all transactions expected to be in <span class='math'>s<sub>actual</sub></span> but not actually in it (excluding those that have been recently broadcast; see below).</p>
+  <p>Block health is a measure of how many transactions appear to be intentionally excluded from a block—a block without any transactions that appear intentionally excluded will have 100% health, while a block with 1 or more transactions that appear intentionally excluded will have sub-100% health.</p>
+  <p>As a result, block health is <i>not</i> intended to be a measure of how closely an expected block resembles an actual block.</p>
+  <p>How is it calculated? Let <span class='math'>s<sub>expected</sub></span> be the set of all transactions in Mempool's expected block and let <span class='math'>s<sub>actual</sub></span> be the set of all transactions in the actual block. Then let <span class='math'>n</span> be the number of all transactions in both <span class='math'>s<sub>expected</sub></span> and <span class='math'>s<sub>actual</sub></span>.</p>
+  <p>Furthermore, let <span class='math'>r</span> be the number of transactions Mempool deduces were <a [routerLink]="['/docs/faq' | relativeUrl]" fragment="how-do-block-audits-work">intentionally excluded</a> from <span class='math'>s<sub>actual</sub></span>.</p>
   <p>Block health is calculated as <span class='math'>n / ( n + r</span> ).</p>
-  <p>Transactions appearing in both <span class='math'>s<sub>expected</sub></span> and <span class='math'>s<sub>actual</sub></span> are used (instead of a block's full transaction count) in order to minimize chances that block health is impacted by missing transactions that don't imply censorship:</p>
+  <p>The number of transactions appearing in both <span class='math'>s<sub>expected</sub></span> and <span class='math'>s<sub>actual</sub></span> is used (instead of a block's full transaction count) in order to minimize chances that block health is inadvertently impacted by transactions that were most likely not intentionally excluded:</p>
   <ul>
     <li>recently-broadcast transactions, since the miner may simply not have received them</li>
     <li>certain low-feerate transactions, since the miner may have opted to replace them with more profitable out-of-band transactions</li>
   </ul>
-  <p>Mempool uses a re-implementation of Bitcoin Core's transaction selection algorithm to determine the transactions it expects to see in the next block.</p>
+  <p>As a result, block health is <i>not</i> intended to be a measure of how closely an expected block resembles an actual block. The actual block can be vastly different from the expected block, but if no transactions appear to be intentionally excluded, it will have a high health rating (<a [routerLink]="['/block/0000000000000000000515e202c8ae73c8155fc472422d7593af87aa74f2cf3d']">extreme example</a>).</p>
```

Comment:
> As a result, block health is <i>not</i> intended to be a measure of how closely an expected block resembles an actual block.

this sentence is repeated verbatim in line 245, I'm not sure if that's unintentional or for deliberate emphasis?

Reply:
Nope, I just forgot to remove that line. Thanks for catching that!